Week 02/Lecture 03/Assignment 05/RemoveDuplicates.java

```diff
@@ -3,29 +3,42 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.util.ArrayList;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
 public class RemoveDuplicates {
     public static void main(String[] args) {
-        String inputFileName = "data/input.csv";
-        String outputFileName = "data/output.csv";
-        String delimiter = ","; // Parse CSV format
-        
-        // Set to store unique keys (employeeID in this case)
+        if (args.length < 3) {
```

Comment:
For best practice, we should create constant for number 3 to prevent magic number. I dont know what is 3 mean ?

Reply:
Hello! thanks for the review
The reason why i should mention at least 3 since the format of I/O on terminal looks something like this
```java
$ java RemoveDuplicates <inputFile> <outputFile> <keyFieldIndex>
```

For example:
```java
$ java RemoveDuplicates data/input.csv data/output.csv 0
```

is it okay to do that? i will create a constant if this is fine.

Reply:
Okay, that's fine, just avoid magic number by naming the variable.

Reply:
@mikeleo03 You can merge this PR to mark as done for week #2. Thanks.

Reply:
Thankyou!